scripts/sim-district/verify-iep-meeting-rls.ts

```
@@ -0,0 +1,259 @@
+    const survived = await storedLocation(meetingId);
+    check(ownerErr?.code === '42501' && survived !== null,
+      'even the ORGANIZER cannot hard-delete (compliance record)',
+      `code=${ownerErr?.code ?? 'none'} row=${survived ? 'intact' : 'GONE'}`);
+
+    const { error: adminErr } = await marcus.client
+      .from('iep_meetings').delete().eq('id', meetingId).select('id');
+    check(adminErr?.code === '42501', 'even a site admin cannot hard-delete',
+      `code=${adminErr?.code ?? 'none'}`);
+  }
+
+  console.log('attendee writes inherit the parent meeting\'s rule:');
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)